### PR TITLE
Do not depend on json template for mocking streaming.

### PR DESCRIPTION
**Pull request by YooSunYoung**
@SimonHeybrock  

This update should allow you to run beamlime only with a file without any json structure.

```bash
beamlime \ 
    --nexus-file-path 60392-2022-02-28_2215.nxs \
    --workflow ess-loki-monitor \
    --log-level DEBUG \
    --merge-every-nth 20 \
    --image-path-prefix loki-test \
    --num-frames 40 \
    #--fill-dummy-data  # uncomment this if you want to use actual data from the file.

```

### CODE REVIEW

File: src/beamlime/applications/handlers.py [hunk 1]
```diff
@@ -105,7 +105,7 @@ def merge_data_piece(self, message: DataPieceReceived) -> DataReady | None:
         self.debug("Data piece merged for %s", message.content.key)
         if self._should_send_message():
             nxevent_data = {
-                nexus_path_as_string(module_spec.path): JSONGroup(value)
```
Comment (YooSunYoung) [inline src/beamlime/applications/handlers.py:108]: There was a bug that I found as I tested it with a file : D....!! wheeee

**Comment by YooSunYoung** [comment]
Feel free to make it better and proceed while I'm on vacation : D ...!

**Comment by YooSunYoung** [comment]
For testing, there are already existing tests that I had updated, but we still need to test ``fill-dummy-data``.
I opened an issue #243 since it was too complicated to do it at once.

**Review by SimonHeybrock — APPROVED** [review]
Worked for testing with the new SANS workflows.

**Pull request merged**